core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/clusterproxy/reregister/ClusterProxyReRegistrationActions.java

```diff
@@ -51,7 +52,7 @@ protected void doExecute(ClusterProxyReRegistrationContext context, ClusterProxy
 
             @Override
             protected Selectable createRequest(ClusterProxyReRegistrationContext context) {
-                if (context.getStack().getTunnel().useCcmV1()) {
+                if (context.getIsCCMv1()) {
```

Comment:
Why is the information on CCM version passed explicitly?  Is "context.getStack().getTunnel().useCcmV1()" not accurate?